tracer/build/_build/Build.Steps.cs

```diff
@@ -1515,189 +1515,247 @@ void RunWindowsIisIntegrationTests(Project project)
+                                .Where(IsNewError)
+                                .ToList();
+
+        var nativeProfilerFiles = logDirectory.GlobFiles("**/DD-DotNet-Profiler-Native-*");
+        var nativeProfilerErrors = nativeProfilerFiles
+                                  .SelectMany(ParseNativeProfilerLogFiles)
+                                  .Where(x => x.Level >= minLogLevel)
+                                  .Where(IsNewError)
+                                  .ToList();
+
+        var nativeLoaderFiles = logDirectory.GlobFiles("**/dotnet-native-loader-*");
+        var nativeLoaderErrors = nativeLoaderFiles
+                                  .SelectMany(ParseNativeProfilerLogFiles) // native loader has same format as profiler
+                                  .Where(x => x.Level >= minLogLevel)
+                                  .Where(IsNewError)
+                                  .ToList();
+
+        var hasRequiredFiles = !allFilesMustExist
+                            || (managedFiles.Count > 0
+                             && nativeTracerFiles.Count > 0
+                             && nativeProfilerFiles.Count > 0
+                             && nativeLoaderFiles.Count > 0);
+
+        if (hasRequiredFiles 
+         && managedErrors.Count == 0 
+         && nativeTracerErrors.Count == 0 
+         && nativeProfilerErrors.Count == 0
+         && nativeLoaderErrors.Count == 0)
+        {
+            Logger.Info("No errors found in managed or native logs");
+            return;
+        }
 
-           static LogLevel ParseManagedLogLevel(string value)
-               => value switch
-               {
-                   "VRB" => LogLevel.Trace,
-                   "DBG" => LogLevel.Trace,
-                   "INF" => LogLevel.Normal,
-                   "WRN" => LogLevel.Warning,
-                   "ERR" => LogLevel.Error,
-                   _ => LogLevel.Normal, // Concurrency issues can sometimes garble this so ignore it
-               };
-
-           static LogLevel ParseNativeLogLevel(string value)
-               => value switch
-               {
-                   "trace" => LogLevel.Trace,
-                   "debug" => LogLevel.Trace,
-                   "info" => LogLevel.Normal,
-                   "warning" => LogLevel.Warning,
-                   "error" => LogLevel.Error,
-                   _ => LogLevel.Normal, // Concurrency issues can sometimes garble this so ignore it
-               };
-
-           Logger.Info($"Skipping log parsing, directory '{logDirectory}' not found");
-       });
+        Logger.Warn("Found the following errors in log files:");
+        var allErrors = managedErrors
+                       .Concat(nativeTracerErrors)
+                       .Concat(nativeProfilerErrors)
+                       .Concat(nativeLoaderErrors)
+                       .GroupBy(x => x.FileName);
+
+        foreach (var erroredFile in allErrors)
+        {
+            Logger.Info();
```

Comment:
Can be removed? Or is it intentional?
```suggestion
```

Reply:
Intentional, to add a blank line between the descriptions, makes them easier to read🙂